src/validator.coffee

```diff
@@ -1,35 +1,54 @@
 class ValidationError extends Error
+  ###
+  @message error message
+  @reason reporting friendly error cause
```

Comment:
Use the notation used in the other project classes (e.g. [biskoto](https://github.com/skroutz/analytics.js/blob/master/src/biskoto.coffee))

```
@param [String] message The error message
@param [Object] reason The reporting friendly error cause
```